core/src/main/java/io/github/tcdl/api/ResponderServer.java

```diff
@@ -34,10 +34,10 @@ private ResponderServer(String namespace, MessageTemplate messageTemplate, MsbCo
     }
 
     /**
-     * Create a new instance of a {@link ResponderServer}.
+     * Create a new instance of {@link ResponderServer}.
      *
-     * @param namespace       topic on a bus for listening incoming requests
-     * @param messageTemplate template which will be used for creating response messages, object of class {@link MessageTemplate}
+     * @param namespace       topic on a bus for listening on incoming requests
+     * @param messageTemplate template used for creating response messages, instance of type {@link MessageTemplate}
```

Comment:
The phrase that goes after comma seems to be redundant. The type is evident from method signature.